sp800-63b/sec6_lifecycle.md

```diff
@@ -18,46 +18,52 @@ The record created by the CSP SHALL contain the date and time the authenticator
+At IAL2, the CSP SHALL bind at least one, and SHOULD bind at least two, authenticators to the subscriber's online identity. Binding of multiple authenticators is preferred in order to recover from loss or theft of their primary authenticator. While at IAL1 all identifying information is self-asserted, creation of online material or an online reputation makes it undesirable to lose control of an account as result of the loss of an authenticator. The second authenticator makes it possible to securely recover from that situation and thus a CSP SHOULD bind at least two authenticators to the subscriber's credential at IAL1 as well.
 
-At IAL 2 and above, identifying information is associated with the online identity and the subscriber has undergone an identity proofing process as described in SP 800-63A. As a result, authenticators at the same AAL as the desired IAL SHALL be bound to the account. For example, if the subscriber has successfully completed proofing at IAL 2, AAL 2 or 3 authenticators are appropriate to bind to the IAL 2 identity. While a CSP MAY bind an AAL 1 authenticator to an IAL 2 identity, if the subscriber is authenticated at AAL 1, the CSP SHALL NOT expose personal information, even if self-asserted, to the subscriber. As above, the availability of additional authenticators provides backup methods of authentication if an authenticator is damaged, lost, or stolen.
+At IAL2 and above, identifying information is associated with the online identity and the subscriber has undergone an identity proofing process as described in SP 800-63A. As a result, authenticators at the same AAL as the desired IAL SHALL be bound to the account. For example, if the subscriber has successfully completed proofing at IAL2, AAL2 or AAL3 authenticators are appropriate to bind to the IAL2 identity. While a CSP MAY bind an AAL1 authenticator to an IAL2 identity, if the subscriber is authenticated at AAL1, the CSP SHALL NOT expose personal information, even if self-asserted, to the subscriber. As above, the availability of additional authenticators provides backup methods of authentication if an authenticator is damaged, lost, or stolen.
 
 Enrollment and binding MAY be broken up into a number of separate physical encounters or electronic transactions. Two electronic transactions are considered to be separate if they are not part of the same protected session. In these cases, the following methods SHALL be used to ensure that the same party acts as applicant throughout the processes:
```

Comment:
Should now be corrected into a structure similar to the previous list, but using a, b, and c at the 2nd level instead of numbers again.